dev/js/wow_wizard.js

```diff
@@ -66,11 +66,71 @@
 					buttonTextColor: '#FFF',
 					imageChoiceBorderColor: "#4068D6",
 					imageChoiceCircleBackgroundColor: "#A2B5EB"
+				},
+        peach: {
+					activeIndicatorBackgroundColor: '#DB764F',
+					activeIndicatorTextColor: '#FFF',
+					questionAndAnswerTextColor: '#7A2851',
+					circleColor: '#7A2851',
+					outlineColor: '#DB764F',
+					lineColor: '#DB764F',
+          activeButtonBackgroundColor: '#DB764F',
+          passiveButtonBackgroundColor: '#FFF',
+					activeButtonTextColor: '#FFF',
+					passiveButtonTextColor: '#000',
+					buttonTextColor: '#FFF',
+					imageChoiceBorderColor: "#DB764F",
+					imageChoiceCircleBackgroundColor: "#9E1E5B"
+				},
+        cherry: {
+					activeIndicatorBackgroundColor: '#CF3851',
+					activeIndicatorTextColor: '#FFF',
+					questionAndAnswerTextColor: '#F0738A',
+					circleColor: '#F0738A',
+					outlineColor: '#CF3851',
+					lineColor: '#CF3851',
+          activeButtonBackgroundColor: '#CF3851',
+          passiveButtonBackgroundColor: '#FFF',
+					activeButtonTextColor: '#FFF',
+					passiveButtonTextColor: '#000',
+					buttonTextColor: '#FFF',
+					imageChoiceBorderColor: "#CF3851",
+					imageChoiceCircleBackgroundColor: "#F0738A"
+				},
+        passionfruit: {
+					activeIndicatorBackgroundColor: '#872945',
+					activeIndicatorTextColor: '#FFF',
+					questionAndAnswerTextColor: '#5BA300',
+					circleColor: '#5BA300',
+					outlineColor: '#872945',
+					lineColor: '#872945',
+          activeButtonBackgroundColor: '#872945',
+          passiveButtonBackgroundColor: '#FFF',
+					activeButtonTextColor: '#FFF',
+					passiveButtonTextColor: '#000',
+					buttonTextColor: '#FFF',
+					imageChoiceBorderColor: "#872945",
+					imageChoiceCircleBackgroundColor: "#5BA300"
+				},
+        coconut: {
+					activeIndicatorBackgroundColor: '#5CD6FF',
+					activeIndicatorTextColor: '#FFF',
+					questionAndAnswerTextColor: '#834820',
+					circleColor: '#834820',
+					outlineColor: '#5CD6FF',
+					lineColor: '#5CD6FF',
+          activeButtonBackgroundColor: '#5CD6FF',
+          passiveButtonBackgroundColor: '#FFF',
+					activeButtonTextColor: '#FFF',
+					passiveButtonTextColor: '#000',
+					buttonTextColor: '#FFF',
+					imageChoiceBorderColor: "#5CD6FF",
+					imageChoiceCircleBackgroundColor: "#834820"
 				}
 			};
 			var styles = {
 				material: {
-					borderRadius: '0.2px'
```

Comment:
I felt like the border-radius was barely visible at 0.2px so I increased it, but I'll change it back if that's what you want.